commands/commands.go

```diff
@@ -31,9 +31,13 @@ type Options struct {
 	LocalOnly bool
 }
 
-func New(opts Options) *cobra.Command {
+func New(opts Options, zeitVersionType string) *cobra.Command {
+	name := "zeitgeist"
+	if zeitVersionType == "remote" {
```

Comment:
Do we want to make `"remote"` and `"local"` enum-like consts?

Reply:
ptal